app/(afterLogin)/(deal)/deals/activity/page.tsx

```
@@ -0,0 +1,317 @@
+const PersonMeta: React.FC<{ name: string; deal: string }> = ({
+  name,
+  deal,
+}) => (
+  <div className="text-xs text-slate-500 leading-tight">
+    <div>{name}</div>
+    <div className="text-slate-400">{deal}</div>
+  </div>
+);
+
+
+const TimeStamp: React.FC<{ time: string }> = ({ time }) => (
```

ESLint

[error] Delete `⏎`

(prettier/prettier)